### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by Changron**
1. fix description too long
   
2. make contest detail looks like other pages (add <br>)
   
3. add contest id and name in contest edit page

### CODE REVIEW

**Comment by nickelcheng** [comment]
Congratulation!!! You hit the 200th PR XDD

**Comment by nickelcheng** [comment]
I prefer to use this format: (No "edit contest" anymore)
785 - Rejudge [ See this contest ] [ Go to Contest Archive ]
Just like edit problem page.

**Comment by nickelcheng** [comment]
Can you add customize title for each contest detail page?
Such as "Contest 785" instead of "Contest" all the time.

**Comment by nickelcheng** [comment]
+1 besides comments.

**Comment by Changron** [comment]
fixed

**Comment by nickelcheng** [comment]
gg again
screenshot from 2015-05-26 23 38 48

**Comment by Changron** [comment]
it looks normal here

**Comment by nickelcheng** [comment]
My browser's bug, again...

**Comment by nickelcheng** [comment]
Please also modify title in ```contest/contest.html```

File: contest/templates/contest/editContest.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% load contest_extras %}
 {% block title_name %}
 {% load bootstrap %}
-<title>{{ title }}</title>
+<title>{{ title }}{{ contest }}</title>
```
Comment (nickelcheng) [inline contest/templates/contest/editContest.html]: I prefer to add a space between ```{{ title }}``` and ```{{ contest }}```

**Comment by nickelcheng** [comment]
+1 besides comments.

**Comment by Changron** [comment]
fixed

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**